.travis.yml

```diff
@@ -28,4 +28,7 @@ env:
     - FEEDBACK_MAIL_TO="example@example.org"
 services:
   - postgresql
-script: "bundle exec rspec"
+script:
+  - "bundle exec rspec"
+after_script:
+  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
To `before_script` we need to add:
```yaml
  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
  - chmod +x ./cc-test-reporter
  - ./cc-test-reporter before-build
```

The Travis settings for this repo also need to have the `CC_TEST_REPORTER_ID` env var set. The value for this is provided by CodeClimate.

CodeClimate is not yet reporting on PRs so a [Github integration](https://docs.codeclimate.com/docs/github) needs to be set up.